wagtail/blocks/base.py

```diff
@@ -157,7 +159,7 @@ def get_default(self):
         model definition time (e.g. something like StructValue which incorporates a
         pointer back to the block definition object).
         """
-        return self.normalize(self.meta.default)
+        return self.normalize(getattr(self.meta, "default", None))
```

Comment:
The `default` is defined in `FieldBlock.Meta.default`, so doing `Block().get_default()` would currently raise an error.